### PR TITLE
Document location, lock, shake

### PR DESCRIPTION
**Pull request by imurchie**
Document that we do not support lock (fixes https://github.com/appium/appium/issues/6813), shake (fixes https://github.com/appium/appium/issues/6814), and location (fixes https://github.com/appium/appium/issues/6856).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,10 @@
 
 ## Known issues
 
-* Unable to interact with elements on iPads in Landscape mode (https://github.com/appium/appium/issues/6994)
+* Unable to interact with elements on devices in Landscape mode (https://github.com/appium/appium/issues/6994)
+* `shake` is not implemented due to lack of support from Apple
+* `lock` is not implemented due to lack of support from Apple
+* Setting geo-locaiton not supported due to lack of support from Apple
```
Comment (jlipps) [inline README.md]: `location`

Comment (jlipps) [inline README.md]: you might already have this in mind, but we also want to document this on the migrating-to-xcuitest doc in our main repo

Comment (imurchie) [inline README.md]: Ugh. Wy cna't I spel?

**Comment by imurchie** [comment]
Related PR: https://github.com/appium/appium/pull/7269

**Pull request merged**